lib/src/metta/runner/stdlib.metta

```diff
@@ -1,4 +1,5 @@
-(@doc = 
+; TODO: for some reason get-doc doesn't work for '='
```

Comment:
One parenthesis after return is not correct.

Reply:
Yes, you right. I've fixed this and now `help!` for `=` works.